aten/src/ATen/native_parse.py

```diff
@@ -383,7 +383,14 @@ def run(paths):
                 declaration['return'] = return_arguments if len(output_arguments) == 0 else output_arguments
                 declaration['variants'] = func.get('variants', ['function'])
                 declaration['requires_tensor'] = func.get('requires_tensor', False)
-                declaration['matches_jit_signature'] = func.get('matches_jit_signature', False)
+                declaration['matches_jit_signature'] = func.get('matches_jit_signature', None)
```

Comment:
just let people set it to True.  This is just going to cause merge conflicts for no benefit.

I.e. replace this whole thing with
```
                declaration['matches_jit_signature'] = func.get('matches_jit_signature', True)
```